cpg_utils/hail_batch.py

```diff
@@ -30,7 +36,7 @@ def init_batch(**kwargs):
     )
 
 
-def copy_common_env(job: hb.job.Job) -> None:
+def copy_common_env(job: Union[hb.batch.job.BashJob, hb.batch.job.Job]) -> None:
```

Comment:
BashJob is a type of Job right?

Reply:
Yeah, this was a silly change. The type reference for Job was wrong, but doesn't need to be duplicated for subtypes. Altered in latest commit